pkgs/by-name/op/openfoam-org/package.nix

```diff
@@ -0,0 +1,122 @@
+{
+  stdenv,
+  bash,
+  m4,
+  flex,
+  bison,
+  fftw,
+  gnumake,
+  mpi,
+  scotch,
+  boost,
+  cgal,
+  zlib,
+  fetchFromGitHub,
+  lib,
+  trilinos-mpi,
+  nix-update-script,
+}:
+stdenv.mkDerivation {
+  pname = "openfoam-org";
+  version = "12";
+  src = fetchFromGitHub {
+    owner = "OpenFOAM";
+    repo = "OpenFOAM-12";
+    rev = "refs/tags/version-12";
+    hash = "sha256-1vFia30IzuSOLL5d3oNrRWSr8EUzwti9wOarUUa2SHM=";
+  };
+  meta = with lib; {
+    description = "Open source computational fluid dynamics toolkit";
+    homepage = "https://github.com/OpenFOAM/OpenFOAM-12";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ gusgibbon ];
+    platforms = with platforms; [ "x86_64-linux" ];
+  };
+  passthru.updateScript = nix-update-script { };
+  nativeBuildInputs = [
+    gnumake
+    bash
+    m4
+    flex
+    bison
+  ];
+  buildInputs = [
+    fftw
+    mpi
```

Comment:
```suggestion
```
Builds without. I think this is covered by `mpi.dev` below, no?

Reply:
I believe you are correct, I probably just left that in there when packaging this.

Reply:
`mpi` in `builtInputs` is automatically taken with `getDev` already so you don't need to specify `.dev`, and it's better not to specify so that we don't enforce that the output should necessarily be split out